.appveyor.yml

```
@@ -62,9 +62,13 @@ build_script:
 - cmd: >-
     set GO111MODULE=on
 
-    go mod tidy
+    pushd . && cd sdk && go mod tidy && go mod vendor && popd
```

```suggestion
    pushd sdk && go mod tidy && go mod vendor && popd
```

Same below, and in `.github/workflows/windows-build.yml`.